# HG changeset patch
# User Vytautas Janƒçauskas <unaudio@gmail.com>
# Date 1379626626 -10800
#      Fri Sep 20 00:37:06 2013 +0300
# Node ID da6173d4e384c29ac88f974d811a459b325fd972
# Parent  3954d835436ba3c91cb3d0137360c7fe27621fc9
doc: included docstrings for audioinfo, audioread and audiowrite in audio.txi

diff --git a/doc/interpreter/audio.txi b/doc/interpreter/audio.txi
--- a/doc/interpreter/audio.txi
+++ b/doc/interpreter/audio.txi
@@ -17,16 +17,18 @@
 @c <http://www.gnu.org/licenses/>.
 
 @c Written by Kurt Hornik <Kurt.Hornik@wu-wien.ac.at> on 1996/05/14
 
 @node Audio Processing
 
 @chapter Audio File Utilities
 
+The following functions allow you to read, write and retrieve information about audio files. Various formats are supported including wav, flac and ogg vorbis.
+
 @DOCSTRING(audioinfo)
 @DOCSTRING(audioread)
 @DOCSTRING(audiowrite)
 
 @chapter Audio Device Information
 
 @DOCSTRING(audiodevinfo)
 
